readme.md

```diff
@@ -10,6 +10,8 @@ It currently runs on Gonk based devices and on desktop platforms (tested on Linu
 
 It is in very early stages and not really usable yet. There is an incomplete [todo list](todo.md) if you want to help!
 
+You can check the **_tutorial-build_** file in the **_docs_** folder to have a more detailed explanation of how to build and run B2G/Servonk.
```

Comment:
Use a link: "You can check the [build tutorial](docs/build-tutorial.md) for a detailed explanation...